README.Rmd

```diff
@@ -51,18 +54,18 @@ This will result in the demonstration REST API running at http://localhost/irods
 
 To connect to the REST API endpoint of your choice, load `rirods`, connect with `create_irods()`, and authenticate with your iRODS credentials:
 
-```{r admin}
+```{r project}
 # load
 library(rirods)
 
 # connect
-create_irods("http://localhost/irods-rest/0.9.3", overwrite = TRUE)
+create_irods("http://localhost/irods-rest/0.9.3")
+```
 
+```{r admin, include=FALSE}
 # authenticate
 iauth("rods", "rods")
-```
 
-```{r admin, include=FALSE}
```

Comment:
I had pulled it out of the admin chunk so that a new reader could see how to load/connect/authenticate.    But this is fine - bobby is covered below...   let's get it in.

Reply:
I thought that this would be more an admin thing that is not covered by the package, as there are no admin functions.

Reply:
It's only 'admin' in that the user logging in is a rodsadmin, it's still just an iauth() call.

All good.  We did it!